### DISCUSSION TITLE
Nvidia Shield - PKC with Plex Server and Kodi on same machine

### DISCUSSION THREAD

**Discussion by OGTK423** [croneter/PlexKodiConnect#1851] [Q&A] [answered]
I run a Plex server hosted on an Nvidia Shield with media supplied from direct attached storage connected to the Shield itself.

Can I install Kodi on this same Shield device and use PlexKodiConnect to have Kodi direct/locally play media from the Plex server?

**Comment by croneter** [comment, marked as answer]
Of course! You could even opt for Direct Paths without having to configure them!

**Reply by OGTK423** [reply]
Fantastic, thanks so much for the response! Definitely interested in being able to always play locally and utilize Direct Paths.

This set up would allow you to use external audio tracks since it would be utilizing Direct Paths, correct?

**Comment by croneter** [comment]
I guess so, just try it 😉
